From 56a109e92e847277b550b54ecf2523bdb775e9cd Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Wed, 2 Nov 2016 08:06:17 +0000
Subject: [PATCH 4/4] context: Only use -mavx2 for AVX2 builds

Peter encountered some issues when using combined flags, so we'll only set
-mavx2 and allow gcc to work out the rest.

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 ypkg2/ypkgcontext.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ypkg2/ypkgcontext.py b/ypkg2/ypkgcontext.py
index 0d4128c..1c31415 100644
--- a/ypkg2/ypkgcontext.py
+++ b/ypkg2/ypkgcontext.py
@@ -27,7 +27,7 @@ PGO_USE_FLAGS = "-fprofile-use -fprofile-dir=\"{}\" -fprofile-correction"
 BIND_NOW_FLAGS = ["-Wl,-z,now"]
 
 # AVX2
-AVX2_FLAGS = "-msse3 -mssse3 -msse4.1 -msse4.2 -mavx -mavx2"
+AVX2_FLAGS = "-mavx2"
 
 
 class Flags:
-- 
2.10.1

